meta/send.rb

```diff
@@ -307,6 +307,70 @@
   mutation 'foo.__send__(self)'
 end
 
+Mutant::Meta::Example.add :send do
+  source 'foo.__send__(:bar)'
+
+  singleton_mutations
+  mutation 'foo.bar'
+  mutation 'foo.__send__'
+  mutation 'foo.public_send(:bar)'
+  mutation 'foo'
+  mutation ':bar'
+  mutation 'self.__send__(:bar)'
+  mutation 'foo.__send__(nil)'
+  mutation 'foo.__send__(self)'
+  mutation 'foo.__send__(:bar__mutant__)'
+end
+
+Mutant::Meta::Example.add :send do
+  source 'foo.send(:bar)'
+
+  singleton_mutations
+  mutation 'foo.bar'
+  mutation 'foo.send'
+  mutation 'foo.__send__(:bar)'
+  mutation 'foo.public_send(:bar)'
+  mutation 'foo'
+  mutation ':bar'
+  mutation 'self.send(:bar)'
+  mutation 'foo.send(nil)'
+  mutation 'foo.send(self)'
+  mutation 'foo.send(:bar__mutant__)'
+end
+
+Mutant::Meta::Example.add :send do # rubocop:disable Metrics/BlockLength
+  source 'foo.public_send(:bar, 1, two: true, **kwargs, &block)'
```

Comment:
I can simplify this if preferred, but I wanted to make sure I didn't somehow mangle a complex send case when i was hacking on this locally.